src/Entities/Weapon.ts

```diff
@@ -5,7 +5,7 @@ import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
 @Entity()
 export class Weapon {
 	@PrimaryGeneratedColumn()
-	id: number;
+	id?: number;
```

Comment:
Can we update CLI's `enity` template to include this update as well?

`project-cli/helper_scripts/Templates/Repositories/Entity.mustache`